app/util/planParamUtil.js

```diff
@@ -198,13 +206,18 @@ export function planQueryNeeded(
 
     case PLANTYPE.SCOOTERTRANSIT:
       /* special logic: relaxed scooter query is made only if no networks allowed */
+      /* special logic: scooter queries are not made if the search is n minutes or more into the future */
       return (
         config.transportModes.scooter.availableForSelection &&
         transitModes.length > 0 &&
         !wheelchair &&
         (relaxSettings
           ? settings.scooterNetworks.length === 0
-          : settings.scooterNetworks.length > 0)
+          : settings.scooterNetworks.length > 0) &&
+        moment(parseDateTime(arriveBy, time).earliestDeparture).diff(
+          moment(),
+          'minutes',
+        ) < config.vehicleRental.maxMinutesToRentalJourneyStart
```

Comment:
Does this work correctly when arriveBy== true?  Code seems to compare moment(undefined) with current time. Maybe arriveBy case needs different logic. If time is tomorrow, scooters surely should not be used.

Also, use of  parseDateTime here seems totally unnecessary as that only converts unix time  to a string and then you convert it  back to moment object.  Please use Date.now() to get epoch ms and compare it to time prop which is  epoch seconds using simple math computation. We are trying to get rid of moment.

Reply:
ArriveBy was accidentally overlooked. Also forgot that we are getting rid of moment. The arriveBy limit is a little hard to define as a journey could take a long time but a scooter journey might be relevant in the beginning, although there is no guarantee that the suggestion would be in the beginning of the journey. I added another configuration for the arriveBy time constraint.